packages/dds/cell/src/cell.ts

```diff
@@ -91,15 +127,26 @@ export class SharedCell<T = any> extends SharedObject<ISharedCellEvents<T>> impl
 
     private readonly pendingMessageIds: number[] = [];
 
+    /**
+     * This enables the cell to store the attribution information which can be accessed with the runtime
+     * (i.e. who creeated the content and when it was created)
+     *
+     * default: false
+     */
+    private readonly trackAttribution: boolean;
+
+    private readonly attributor: CellAttributor = new CellAttributor();
+
     /**
      * Constructs a new `SharedCell`.
      * If the object is non-local an id and service interfaces will be provided.
      *
      * @param runtime - The data store runtime to which the `SharedCell` belongs.
      * @param id - Unique identifier for the `SharedCell`.
      */
-    public constructor(id: string, runtime: IFluidDataStoreRuntime, attributes: IChannelAttributes) {
+    public constructor(id: string, runtime: IFluidDataStoreRuntime, attributes: IChannelAttributes, trackAttribution?: boolean) {
```

Comment:
This should be consistent with the option format that merge-tree uses. Something like this:

```
        const mergeTreeOptions = {
            ...dataStoreRuntime.options as IMergeTreeOptions
        };

        const configSetAttribution = loggerToMonitoringContext(this.logger).config.getBoolean("Fluid.Attribution.EnableOnNewFile");
        if (configSetAttribution !== undefined) {
            mergeTreeOptions.attribution ??= {};
            mergeTreeOptions.attribution.track = configSetAttribution;
        }
```

We should probably have a helper for extracting this setting correctly from config and data store runtime options, but for now duplicating the code but keeping it consistent is better than adding a separate configuration mechanism.